api/proto/attest/attest.proto

```diff
@@ -14,6 +14,7 @@ message ZAttestReq {
   ZAttestReqType reqType = 1;  //type of the request
   ZAttestQuote quote = 2;      //attestation quote msg
   repeated ZEveCert certs = 3; //X509 certs in .PEM format, signed by device certificate
```

Comment:
Please add "to be deprecated" for this as well.

Reply:
added the necessary comment line